bin/sqlitedb.py

```diff
@@ -1,5 +1,6 @@
 import csv, sqlite3, re, sys
 from functools import reduce
+from txt2params import gen_params
```

Comment:
Looks like we don't need it here more.

Reply:
We did not need it before but now that metadata_json lives in sqlitedb.py,  and it calls  gen_params(), we need to keep it there.